### ISSUE TITLE
Get the "Unknown custom element" when running test on the component using <script setup> in Vue 2.7

### ISSUE THREAD

**Issue by Ceall8650** [vuejs/vue-jest#496]
Hi everyone

I write my component with composition API in Vue 2.7, and I also use `<script setup>` in my component.
When I run the test, I got the warning message in the console

```js
console.error
    [Vue warn]: Unknown custom element: <CustomComponent> - did you register the component correctly? For recursive components, make sure to provide the "name" option.
    
    found in
    
    ---> <Anonymous>
           <Root>

      3 |
      4 | describe('CustomComponent component', () => {
    > 5 |       const wrapper = mount(CustomComponent);
        |                       ^
      6 |
      7 |       it('renders component correctly', () => {
      8 |               expect(wrapper.element).toMatchSnapshot();
```
Does anyone have encountered the similar issue and solve it successful?  

My component is more like below
```js
// MyComponent.vue

<template>
	<div>
		<CustomComponent />
	</div>
</template>

<script setup>
import CustomComponent from './CustomComponent.vue';

<script>
```

test case
```js
import { mount } from '@vue/test-utils';
import CustomComponent from 'pages/CustomComponent.vue';

describe('CustomComponent component', () => {
	const wrapper = mount(CustomComponent);

	it('renders component correctly', () => {
		expect(wrapper.element).toMatchSnapshot();
	});
});

```
Environment
--
vue: ^2.7.10
@vue/vue2-jest: ^29.0.0
babel-jest: 29.0.3
@vue/test-utils: ^1.3.0

**Comment by C-Westbrook** [comment]
try <custom-component />

**Comment by lmiller1990** [comment]
Try 29.1.0 https://github.com/vuejs/vue-jest/releases/tag/v29.1.0 a PR was merged hopefully fixing Vue 2.7 and script setup.

**Comment by Ceall8650** [comment]
@lmiller1990 It works! Thanks!
